examples/demo.go

```diff
@@ -12,6 +12,7 @@ func main() {
 	distro := wsl.Distro{Name: "Ubuntu-22.04-test"}
 
 	// Registering a new distro
+	// TODO: install ubuntu-preview from the store?
```

Comment:
We use the rootfs directly, no need for any application other that WSL.

Reply:
I added it to the CONTRIBUTING document in another comment, I'll link it here as well.
The script used to download the rootfs is the same as in the CI so it should stay up to date.

On the other hand, we can use:
```powershell
Copy-Item `
    "$((Get-AppPackage | Where-Object Name -like 'CanonicalGroupLimited.Ubuntu').InstallLocation)\install.tar.gz" `
    ".\images\jammy.tar.gz"
```

Btw this problem also applies to the integration tests. We may want to simply create a script (à la Makefile, but we're not in Linux 😅) that prepares the repo instead of instructing users on how to do it manually.

Reply:
Anyways, I can address this in a follow-up PR.

Reply:
Ack, I agree that for running the integration tests, we should at least mention we need a rootfs available.

Reply:
(and I have committed your suggestion and will let you update it in  a follow-up PR)